addon/utils/query-cache.js

```diff
@@ -22,6 +22,10 @@ export default class QueryCache {
     return this._internalCache[this._getKey(type, key)];
   }
 
+  has(type, key) {
```

Comment:
Should we call this `contains`? This seems to be fairly conventional for a cache API.